hoomd/md/methods.py

```diff
@@ -339,6 +409,37 @@ def __preprocess_stress(self,value):
         else:
             return (value,value,value,0,0,0)
 
+    def thermalize_extra_dof(self, seed):
```

Comment:
The meaning of `extra` is more obvious here than in the C++ because the docstring is right there; maybe it's OK as is, I'd still prefer a more meaningful name but I wouldn't waste too long on it if you don't like my suggestion and can't think of a better one.

Reply:
I'm open to suggestions for better names. "Extra" is a catch-all that applies to both the thermostat and barostat. I didn't put too much thought into it. Reservoir works for a thermostat but I'm not sure that is the most natural name for the barostat. Maybe "additional"? Would be best if it isn't so generic. Will think on it.

Reply:
@b-butler any thoughts?

Reply:
We could be really explicit and avoid fluid representations using `NVT.thermalize_thermostat_dof` and `NPT.thermalize_thermostat_and_barostat_dof`. That matches the names of the parameter attriburtes.